test/cypress/plugins/index.js

```diff
@@ -18,30 +18,210 @@ module.exports = (on, config) => {
 
   // The same fixed user config is used to generate pacts/contracts, so generating cookies from this config
   // ensures our pact-stub server is ready to return canned responses.
-  const ssUserConfig = require('../../fixtures/config/self_service_user.json')
+  const selfServiceUserConfig = require('../../fixtures/config/self_service_user.json')
 
-  const ssUser = ssUserConfig.config.users.filter(fil => fil.isPrimary === 'true')[0]
+  // Default user
+  const selfServiceUser = selfServiceUserConfig.config.users.filter(fil => fil.isPrimary === 'true')[0]
```

Comment:
I think you could you use `.find()` rather than `.filter()[0]`. `find()` will return the first matching record otherwise `undefined`.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
Good point! I addressed it in https://github.com/alphagov/pay-selfservice/pull/1072/commits/424ccd9582f7785eb396e0b1fe02e39149011742

We are using `.filter()[0]` a lot in our code in other places, so probably we should refactor it at some point as well